src/contrib/cluster/Akka.DistributedData/Serialization/ReplicatorMessageSerializer.cs

```diff
@@ -409,7 +409,7 @@ private Proto.Msg.Status StatusToProto(Status status)
                 proto.Entries.Add(new Proto.Msg.Status.Types.Entry
                 {
                     Key = entry.Key,
-                    Digest = ByteString.CopyFrom(entry.Value.ToByteArray())
```

Comment:
ByteString is an immutable class, there is no need to copy it into yet another byte array.